applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py

```diff
@@ -116,6 +116,16 @@ def GetDefaultParameters(cls):
             "line_search": false,
             "use_old_stiffness_in_first_iteration": false,
             "compute_reactions": true,
+            "solving_strategy_settings": {
+                "strategy_type" : "newton_raphson",
+                "advanced_settings" : {
+                    "desired_iterations" : 4,
+                    "max_radius_factor" : 10.0,
+                    "min_radius_factor" : 0.1,
+                    "loads_sub_model_part_list": [],
+                    "loads_variable_list" : []
+                }
```

Comment:
```suggestion
                "advanced_settings" : { }
```

this should be empty for two reasons:
1. these parameters are invalid for other strategies and will fail during validation
2. the actual strategy should take care of its best default, here it is just copy pasted and can easily be forgotten if sth gets changed